scripts/gfg.js

```diff
@@ -53,13 +51,14 @@ function findDifficulty() {
 }
 
 function getProblemStatement() {
-  const ele = document.getElementsByClassName('problem-statement')[0];
+  const ele = document.querySelector('[class^="problems_problem_content"]');
   return `${ele.outerHTML}`;
 }
 
 function getCode() {
-  let hackyScriptContent = `
-  console.log("trying to get editor content");
+  const hackyScriptContent = `
```

Comment:
delete this

Reply:
Sorry didn't understand, could you please share more info.
As this code gets appended in the script, here :point_down: 
https://github.com/ASahu16/LeetHub/blob/a36191a8ac39356a80d55b90690fbf1cfede774e/scripts/gfg.js#L74-L76

Reply:
1. rename `hackyScriptContent` to `scriptContent`
2. delete the print statement.

Reply:
Addressed